src/Core/Domain/Order/QueryResult/OrderInvoiceAddressForViewing.php

```diff
@@ -194,6 +202,14 @@ public function getCityName(): string
         return $this->cityName;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getDni(): ?string
```

Comment:
Best to avoid mixed types when possible.

```suggestion
     * @return string
     */
    public function getDni(): string
```